r4/src/main/java/org/opencds/cqf/r4/builders/RequestGroupActionBuilder.java

```diff
@@ -40,6 +42,24 @@ public RequestGroupActionBuilder buildDocumentation(List<RelatedArtifact> docume
         return this;
     }
 
+    public RequestGroupActionBuilder buildRelatedAction(
+            List<PlanDefinition.PlanDefinitionActionRelatedActionComponent> planDefinitionRelatedActions) {
+        List<RequestGroup.RequestGroupActionRelatedActionComponent> requestGroupRelatedActions = new ArrayList<RequestGroup.RequestGroupActionRelatedActionComponent>();
```

Comment:
Not worth the time to do a whole new commit to fix, but for future reference this is a great opportunity to use the "diamond operator", where basically you don't have to specify the generic type on the right-hand side of an expression if it's defined on the left hand side. 

```suggestion
        List<RequestGroup.RequestGroupActionRelatedActionComponent> requestGroupRelatedActions = new ArrayList<>();
```